boards/others/esp32c3_supermini/esp32c3_supermini.dts

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2024 Espressif Systems (Shanghai) Co., Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <espressif/esp32c3/esp32c3_mini_n4.dtsi>
+#include "esp32c3_supermini-pinctrl.dtsi"
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+#include <zephyr/dt-bindings/led/led.h>
+#include <espressif/partitions_0x0_default.dtsi>
```

Comment:
Default partition scheme is using the 4MB flash, if esp32c3_supermini uses different flash size please update the included file here.

Reply:
I haven't seen any with less than 4mb.

Reply:
I've seen some C3 and S2 boards which have only 2MB flash, therefore I mentioned it.